static/app/views/settings/projectGeneralSettings.spec.jsx

```diff
@@ -362,81 +309,59 @@ describe('projectGeneralSettings', function () {
           slug: 'new-project',
         },
       });
-      wrapper = mountWithTheme(
+
+      render(
         <ProjectContext orgId={org.slug} projectId={project.slug}>
           <ProjectGeneralSettings
             routes={[]}
             location={routerContext.context.location}
             params={params}
           />
         </ProjectContext>,
-        routerContext
+        {context: routerContext}
       );
     });
 
-    afterEach(() => {
-      wrapper?.unmount();
-      modal?.unmount();
-    });
-
     it('can cancel unsaved changes for a field', async function () {
-      await act(tick);
-      wrapper.update();
+      expect(screen.queryByRole('button', {name: 'Cancel'})).not.toBeInTheDocument();
 
-      // Initially does not have "Cancel" button
-      expect(wrapper.find('Alert button[aria-label="Cancel"]')).toHaveLength(0);
-      // Has initial value
-      expect(wrapper.find('input[name="resolveAge"]').prop('value')).toBe(19);
+      const autoResolveSlider = getField('slider', 'Auto Resolve');
+      expect(autoResolveSlider).toHaveValue('19');
 
       // Change value
-      wrapper
-        .find('input[name="resolveAge"]')
-        .simulate('input', {target: {value: 12}})
-        .simulate('mouseUp');
-
-      // Has updated value
-      expect(wrapper.find('input[name="resolveAge"]').prop('value')).toBe(12);
-      // Has "Cancel" button visible
-      expect(wrapper.find('Alert button[aria-label="Cancel"]')).toHaveLength(1);
+      fireEvent.change(autoResolveSlider, {target: {value: '12'}});
+      expect(autoResolveSlider).toHaveValue('12');
 
       // Click cancel
-      wrapper.find('Alert button[aria-label="Cancel"]').simulate('click');
-      wrapper.update();
+      userEvent.click(screen.getByRole('button', {name: 'Cancel'}));
+      await act(tick);
```

Comment:
do we need this `tick` ? can we use some of the async queries like `findBy` or `waitFor`?

Reply:
there was something that made this hard. But i don't know what.

I think in the future we can do a tick / act audit, since there shouldn't be TOO many, and clean that all up